test/native/src/test/java/org/apache/shardingsphere/test/natived/jdbc/transactions/base/SeataTest.java

```diff
@@ -75,6 +81,9 @@ void assertShardingInSeataTransactions() throws SQLException {
         initEnvironment();
         testShardingService.processSuccess();
         testShardingService.cleanEnvironment();
+        try (Connection connection = dataSource.getConnection()) {
+            connection.unwrap(ShardingSphereConnection.class).getContextManager().close();
+        }
```

Comment:
- I don't really know if there is an easier way to trigger a call to `org.apache.shardingsphere.transaction.base.seata.at.SeataATShardingSphereTransactionManager#close()` than `java.sql.Connection.unwrap(ShardingSphereConnection.class).getContextManager().close();`. What do you think?